src/content/developers/docs/nodes-and-clients/nodes-as-a-service

```diff
@@ -0,0 +1,95 @@
+---
+title: Nodes as a Service
+description: An entry-level overview of Node Services, the pros and cons, and popular providers.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+## Introduction {#Introduction}
+
+Running your own [Ethereum node](/developers/docs/nodes-and-clients/#what-are-nodes-and-clients) can be challenging, especially when getting started or while scaling fast. There are a [number of services](/developers/docs/nodes-and-clients-nodes-as-a-service/#popular-node-services) that run optimized node infrastructures for you, so you can focus on developing your application or product instead. We'll explain how node services work, the pros and cons for using them and list providers if you are interested in getting started. 
```

Comment:
The link in this line won't work. I also think for anchor links in the page, you just need the ID, not the whole link.

`[number of services](#popular-node-services)` should work

Reply:
When you say the link in this line won't work, which link are you referring to? Just the anchor link?

Reply:
it's the anchor link because there's a missing `/` between `nodes-and-clients` and `nodes-as-a-service` – but I think it should work just as the [number of services](#popular-node-services)